app/components/billing-status-badge.ts

```diff
@@ -22,10 +22,46 @@ export default class BillingStatusDisplayComponent extends Component<Signature>
     return this.currentUser?.activeDiscountForYearlyPlan || null;
   }
 
+  get activeDiscountForYearlyPlanExcludingStage2CompletionDiscount(): PromotionalDiscountModel | null {
+    return this.currentUser?.activeDiscountForYearlyPlanExcludingStage2CompletionDiscount || null;
+  }
+
+  get badgeType(): 'vip' | 'member' | 'discount-timer' | 'discount-timer-excluding-stage-2-completion' | 'free-weeks-left' | 'upgrade' {
+    if (this.shouldShowVipBadge) {
+      return 'vip';
+    }
+
+    if (this.shouldShowMemberBadge) {
+      return 'member';
+    }
+
+    if (this.activeDiscountForYearlyPlan && this.canSeeDiscountCountdown) {
```

Comment:
```suggestion
    if (this.activeDiscountForYearlyPlan && this.featureFlags.canSeeDiscountCountdown) {
```

This is one option, another would be to do `this.shouldShowDiscountCountdown` - that has the downside where the logic for each will not be placed together. Doing that split might make more sense once we end the stage 2 experiment